scripts/screener/screener.runner.ts

```diff
@@ -48,23 +56,15 @@ async function scheduleScreenerBuild(
     }),
   });
 
-  if (response.status !== 201) {
-    throw new Error(`Call to proxy failed: ${response.status}`);
+  if (response.status === 200) {
+    console.log('Skipping screener check');
   }
-  //checkUrl
-  return response.json().then(url => url);
-}
 
-async function notifyIntegration(payload: ScreenerProxyPayload) {
-  const fetchResponse = await fetch(environment.screener.proxyUri, {
-    method: 'post',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify(payload),
-  });
-
-  if (fetchResponse.status !== 200) {
-    throw new Error(`Notify integration failed: ${fetchResponse.status}`);
+  if (response.status !== 201 && response.status !== 200) {
+    throw new Error(`Call to proxy failed: ${response.status}`);
   }
+  //conclusion of the screener run
+  return (await response.json()) as ScheduleScreenerBuildResponse;
```

Comment:
we dont need await again  no ?

```suggestion
  return response.json() as ScheduleScreenerBuildResponse;
```